tier/helm/tier/deps.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  application: true
+  description: Deploys tier crafted for the target cloud
+spec:
+  dependencies:
+    - type: helm
+      name: bootstrap
+      repo: bootstrap
+      version: ">= 0.5.1"
+    - type: terraform
+      name: aws
+      repo: tier
+      version: ">= 0.1.0"
+      optional: true
```

Comment:
**suggestion:** as there are provider specific terraform there should be a single module named `kube` that creates the namespace. Rename `aws` to `kube`, make it mandatory by deleting `optional: true` and remove other provider `azure` and `gcp`